spec/salestation/web_spec.rb

```diff
@@ -36,5 +36,15 @@ def status(*)
         expect(web_app.process(response)).to eql(JSON.dump(response_body.merge(debug_message: '')))
       end
     end
+
+    context 'when no content response' do
+      let(:response) {
+        Salestation::Web::Responses.to_no_content.call
```

Comment:
changing this to `to_ok` should also pass... Also, maybe write a unit test instead. easier to test. this doesn't look correct place for it.

Reply:
https://github.com/salemove/salestation/pull/20/commits/06307f32c449904c04f00c0dd0f0688e6eafe074